static/bidder-info/axis.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "http://prebid.axis-marketplace.com/{{.AccountID}}?token={{.SourceId}}"
+maintainer:
+  email: "help@axis-marketplace.com"
```

Comment:
@PyjamaWarrior I have sent an email to this mail address for maintainer email verification. Could you please reply to that mail to complete the verification. It is a required step before onboarding a new adapter.